pkg/alert/alert.go

```diff
@@ -1,8 +1,177 @@
 package alert
 
-import "fmt"
+import (
+    "fmt"
+    "os"
+    "time"
+    "encoding/json"
+    "io/ioutil"
+    "net/http"
 
-func Stale() error {
-	fmt.Println("FIXME!")
+    "github.com/anynines/a9s-aha-cli/pkg/notify"
+
+    log "github.com/sirupsen/logrus"
+)
+
+type Content struct {
+    Idea []struct {
+        Name string `json:"reference_num"`
+        Workflow_status struct {
+            Name string `json:"name"`
+        } `json:"workflow_status"`
+    } `json:"ideas"`
+}
+
+type Req struct {
+    Comments struct {
+        Updated_at string `json:"updated_at"`
+        Name string `json:"name"`
+    } `json:"idea"`
+}
+
+func Stale(notifyFlag bool) (error) {
+    var status Content
+    var updated_at Req
+    var msg string
+
+    err := CheckEnvVariables(notifyFlag)
+    if err != nil {
+        return err
+    }
+
+    log.Info("Query aha.io idea list: ")
+    err = fetchData(&status)
+    if err != nil {
+        return err
+    }
+
+    for i, _ := range status.Idea {
+        tmp, _ := CheckStale(status, updated_at, i, notifyFlag)
+        msg += tmp
+    }
+
+    printReport(notifyFlag, msg)
+    return nil
+}
+
+func CheckStale(status Content, updated_at Req, i int, notifyFlag bool) (string, error) {
+    if status.Idea[i].Workflow_status.Name == "Needs review" {
```

Comment:
The code is easier to read if it's not too much nested. I think it's better to use a guard close (return stmt) here.
```
if status != needs review {
  return "", nil
}

// the actual code
```